DolbyIO/Source/Private/Utils/DolbyIOConversions.cpp

```diff
@@ -156,6 +156,9 @@ namespace DolbyIO
 
 	spatial_audio_style ToSdkSpatialAudioStyle(EDolbyIOSpatialAudioStyle SpatialAudioStyle)
 	{
+#if PLATFORM_ANDROID
+		return spatial_audio_style::disabled;
+#endif
```

Comment:
I don't know. The only thing I know is that when I was loading the game and was joining the conference, a crash was happening in the set spatial position at the dvc level. Which we don't use if I'm not mistaken in the android sdk and I can't really debug that without the symbols from dvc. But maybe it works or maybe something else is happening.

It'll depend which one of us is quicker on that regard :p

Reply:
It didn't work for me either, but I didn't look into the crash at all really.

Reply:
One other question that comes to mind: is there an Android equivalent of the desktop_app/showcase_app for desktop platforms? As in, a sample app that uses the C++SDK? desktop_app is very useful on desktops because it helps determine whether an issue is in plugin, app or SDK.

Reply:
we have a project for the actual android sdk. The interesting part is that I don't remember we even use the dvc's set position even in the regular project. Worth investigating sometime later for sure